2022-09-september/README.md

```diff
@@ -2,19 +2,23 @@
 
 Hey friends!
 
-<img src="https://ca.slack-edge.com/T0266FRGM-USNPNJXNX-1194588c3884-48" width="16"> [Sam](https://github.com/sampoder), Hack Clubber from Singapore, here. It's been a busy couple of weeks in the Hack Club community; I wanted to email all of you with a rundown of what your fellow Hack Clubbers have been up to this past month! 
+<img src="https://ca.slack-edge.com/T0266FRGM-USNPNJXNX-1194588c3884-48" width="16"> [Sam](https://github.com/sampoder), Hack Clubber from Singapore, here. 
+
+This summer, 183 Hack Clubbers came to SF for 42 hour-hackathon- see our new documentary [here](https://www.youtube.com/watch?v=PnK4gzO6S3Q).
```

Comment:
This feels out of place and I bit forced

Reply:
what about smth like?

Sam, Hack Clubber from Singapore, here.

This summer, I helped bring 183 Hack Clubbers to SF for a 42-hour hackathon; a short documentary on this is finally out! Watch it here.